main.go

```diff
@@ -74,6 +104,21 @@ func main() {
 			w.Write([]byte("OK"))
 		})
 
+		pulse := pulse.NewConnection(
```

Comment:
we should only create and attempt to listen for pulse if there is a pulse configuration.